src/view/dataElements/xdmObject/components/noSelectedNodeView.styl

```diff
@@ -1,3 +0,0 @@
-.NoSelectedNodeView-description {
```

Comment:
I noticed this was removed, but not its usage. It's probably worth removing its usage as well.